xml/Microsoft.Extensions.Caching.Memory/MemoryCacheOptions.xml

```diff
@@ -210,9 +210,11 @@
         <summary>Gets or sets the maximum size of the cache.</summary>
         <value>To be added.</value>
         <remarks>
-          <format type="text/markdown"><![CDATA[
-> [!CAUTION]
-> There may be transient states while the cache is full but blocks are being released on a background thread. In this state, unexpected behavior might occur if the cache is being overutilized. 
+          <format type="text/markdown"><![CDATA[
+> [!CAUTION]
+> There may be transient states while the cache is full but blocks are being released on a background thread. In this state, unexpected behavior might occur if the cache is being overutilized.
+>
+> When updating an existing item with another one that will make cache size go beyond the limit, the existing and new items will be removed.
```

Comment:
I don't know that this makes sense. Isn't "updating an existing item with another one" actually replacing an item? The wording sounds like "replace" not "update"